PayItForward.Data.Services/DonationsService.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace PayItForward.Services
+{
+    using System;
+    using AutoMapper;
+    using PayItForward.Data;
+    using PayItForward.Data.Models;
+    using PayItForward.Models;
+    using PayItForward.Services.Abstraction;
+    using PayItForwardDbmodels = PayItForward.Data.Models;
+
+    public class DonationsService : IDonationsService
+    {
+        private readonly IRepository<PayItForwardDbmodels.Donation, Guid> donationsRepo;
+        private readonly IRepository<PayItForwardDbmodels.Story, Guid> storiesRepo;
+        private readonly IRepository<PayItForwardDbmodels.User, string> usersRepo;
+        private readonly IMapper mapper;
+
+        public DonationsService(
+            IRepository<PayItForwardDbmodels.Donation, Guid> donationsRepo,
+            IRepository<PayItForwardDbmodels.Story, Guid> storiesRepo,
+            IRepository<PayItForwardDbmodels.User, string> usersRepo,
+            IMapper mapper)
+        {
+            this.donationsRepo = donationsRepo;
+            this.storiesRepo = storiesRepo;
+            this.usersRepo = usersRepo;
+            this.mapper = mapper;
+        }
+
+        // TODO: TEST
+        public bool Add(DonationDTO donation, Guid storyId)
+        {
+            var makeDonation = false;
```

Comment:
As this variable is bool type, the name should be something like isDonationSuccessfully, but you will not need this variable because of other comment

Reply:
Done